src/ec/.storybook/webpack.config.js

```diff
@@ -13,8 +17,7 @@ module.exports = ({ config, mode }) => {
   if (mode === 'PRODUCTION') {
     // Remove ProgressPlugin (4th plugin)
     const plugin = config.plugins.splice(3, 1);
-
-    if (!(plugin[0] instanceof webpack.ProgressPlugin)) {
+    if (!(plugin[0].constructor.name === 'ProgressPlugin')) {
```

Comment:
ok, this works too, but it doesn't guarantee you're using the same version of the library :)